vscode-build/vscode.patch

```diff
@@ -1121,148 +1062,75 @@ index b9beba774a5..40627803528 100644
+ 			scope: ConfigurationScope.LANGUAGE_OVERRIDABLE
  		},
- 		'files.watcherExclude': {
-diff --git a/src/vs/workbench/contrib/search/browser/search.contribution.ts b/src/vs/workbench/contrib/search/browser/search.contribution.ts
-index 79e5defa112..318ab13d6fd 100644
---- a/src/vs/workbench/contrib/search/browser/search.contribution.ts
-+++ b/src/vs/workbench/contrib/search/browser/search.contribution.ts
-@@ -889,7 +889,7 @@ configurationRegistry.registerConfiguration({
- 		'search.quickOpen.includeHistory': {
- 			type: 'boolean',
- 			description: nls.localize('search.quickOpen.includeHistory', "Whether to include results from recently opened files in the file results for Quick Open."),
--			default: true
-+			default: false // FIXME We should be filtering history based on project rather than disabling it
- 		},
- 		'search.quickOpen.history.filterSortOrder': {
- 			'type': 'string',
-diff --git a/src/vs/workbench/contrib/url/browser/trustedDomains.ts b/src/vs/workbench/contrib/url/browser/trustedDomains.ts
-index 265a5dc43e3..8f1db546879 100644
---- a/src/vs/workbench/contrib/url/browser/trustedDomains.ts
-+++ b/src/vs/workbench/contrib/url/browser/trustedDomains.ts
-@@ -217,7 +217,11 @@ export function readStaticTrustedDomains(accessor: ServicesAccessor): IStaticTru
+diff --git a/src/vs/workbench/services/extensionManagement/browser/builtinExtensionsScannerService.ts b/src/vs/workbench/services/extensionManagement/browser/builtinExtensionsScannerService.ts
+index e1913359976..6c6e80eeabc 100644
+--- a/src/vs/workbench/services/extensionManagement/browser/builtinExtensionsScannerService.ts
++++ b/src/vs/workbench/services/extensionManagement/browser/builtinExtensionsScannerService.ts
+@@ -51,10 +51,10 @@ export class BuiltinExtensionsScannerService implements IBuiltinExtensionsScanne
+ 			if (builtinExtensionsServiceUrl) {
+ 				let bundledExtensions: IBundledExtension[] = [];
  
- 	const defaultTrustedDomains = [
- 		...productService.linkProtectionTrustedDomains ?? [],
--		...environmentService.options?.additionalTrustedDomains ?? []
-+		...environmentService.options?.additionalTrustedDomains ?? [],
-+		"https://utopia.app",
-+		"https://utopia.fm",
-+		"https://utopia.pizza",
-+		"https://utopia95.com"
- 	];
+-				if (environmentService.isBuilt) {
+-					// Built time configuration (do NOT modify)
+-					bundledExtensions = [/*BUILD->INSERT_BUILTIN_EXTENSIONS*/];
+-				} else {
++				// if (environmentService.isBuilt) {
++				// 	// Built time configuration (do NOT modify)
++				// 	bundledExtensions = [/*BUILD->INSERT_BUILTIN_EXTENSIONS*/];
++				// } else {
```

Comment:
This annoying piece of code was preventing us from slipping our extension into the list of built in extensions

Reply:
This comment and probably some of the others would be helpful if they were in the code.

Reply:
Good shout, I'll add them